csharp/sources/Valkey.Glide.InterOp/Native/ValueUnion.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+
+namespace Valkey.Glide.InterOp.Native;
+
+[EditorBrowsable(EditorBrowsableState.Advanced)]
+[SuppressMessage("ReSharper", "InconsistentNaming")]
+[StructLayout(LayoutKind.Explicit, CharSet = CharSet.Unicode)]
+public struct ValueUnion
+{
+    [FieldOffset(0)]
+    public long i;
+
+    [FieldOffset(0)]
+    public double f;
+
+    [FieldOffset(0)]
```

Comment:
Love that. I missed it in my impl.

Reply:
My process is to copy the Rust code directly into the C# side and manually transpile from there.  
So yeah :D happens